chart/compass/values.yaml

```diff
@@ -124,49 +124,49 @@ global:
       parent: 4c31b7c7-2bea-4bd5-9ea5-e9a8d704f900
   images:
     containerRegistry:
-      path: eu.gcr.io/kyma-project/incubator
+      path: europe-docker.pkg.dev/kyma-project
     connector:
-      dir:
+      dir: dev/incubator/
       version: "PR-3022"
       name: compass-connector
     connectivity_adapter:
-      dir:
+      dir: dev/incubator/
       version: "PR-3022"
       name: compass-connectivity-adapter
     pairing_adapter:
-      dir:
+      dir: dev/incubator/
       version: "PR-3022"
       name: compass-pairing-adapter
     director:
-      dir:
+      dir: dev/incubator/
       version: "PR-3002"
       name: compass-director
     hydrator:
-      dir:
+      dir: dev/incubator/
       version: "PR-3022"
       name: compass-hydrator
     ias_adapter:
-      dir:
+      dir: dev/incubator/
       version: "PR-3022"
       name: compass-ias-adapter
     gateway:
-      dir:
+      dir: dev/incubator/
       version: "PR-3022"
       name: compass-gateway
     operations_controller:
-      dir:
+      dir: dev/incubator/
       version: "PR-3022"
       name: compass-operations-controller
     ord_service:
-      dir:
+      dir: dev/incubator/
       version: "PR-92"
       name: compass-ord-service
     schema_migrator:
-      dir:
+      dir: dev/incubator/
       version: "PR-3022"
       name: compass-schema-migrator
     system_broker:
-      dir:
+      dir: dev/incubator/
       version: "PR-3022"
       name: compass-system-broker
     certs_setup_job:
```

Comment:
This is a corner case. 
The image is missing on the new repo. We need to clarify what will happen with old repo. If it is decommissioned, then we may need to transport this image to the new one.

Reply:
Check the other comment as well but it will be available in the old registry